packages/design-system/src/components/StepList/StepList.jsx

```diff
@@ -74,16 +74,19 @@ StepList.propTypes = {
   /**
    * A template string for the aria-label describing a step's actions where
    * the substring `%{step}` is replaced with that step's `heading`.
+   * To remove `aria-label` from a step's actions,  set `actionsLabelText=""`
```

Comment:
I don't know if this is necessary, most of the time we don't want users to remove the aria-labels, so I think its enough to provide an option without necessarily calling it out that overtly. The fact that it's not marked as required might be enough

Reply:
Yes, that works too.